cmd/jaeger/internal/all-in-one.yaml

```diff
@@ -28,12 +28,9 @@ extensions:
           max_traces: 100000
 
   remote_sampling:
-    # We can either use file or adaptive sampling strategy in remote_sampling
-    file:
-      path: ./cmd/jaeger/sampling-strategies.json
```

Comment:
We still want to use a file based strategy by default. A user running all-in-one should be able to override the path via `--set` flag.